modules/prebidServerBidAdapter/index.js

```diff
@@ -517,6 +517,14 @@ Object.assign(ORTB2.prototype, {
     // transform ad unit into array of OpenRTB impression objects
     let impIds = new Set();
     adUnits.forEach(adUnit => {
+      // TODO: support labels / conditional bids
+      // for now, just warn about them
+      adUnit.bids.forEach((bid) => {
+        if (bid.mediaTypes != null) {
+          logWarn(`Prebid Server adapter does not (yet) support bidder-specific mediaTypes for the same adUnit. Size mapping configuration will be ignored for adUnit: ${adUnit.code}, bidder: ${bid.bidder}`);
+        }
+      })
+
```

Comment:
Can you please provided an example Ad unit setup where we use bidder-specific mediaTypes? I don't understand the meaning of this warning message.

Reply:
@Fawke - this is the 'relevantMediaTypes feature described in https://docs.prebid.org/dev-docs/modules/sizeMappingV2.html